services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/tenant/AutowiredTenantInformationService.java

```diff
@@ -53,6 +55,18 @@
                 });
     }
 
+    @Override
+    public Future<TenantResult<TenantObject>> getTenant(final String tenantId, final Span span) {
+        return service.get(tenantId, span)
+                .flatMap(result -> {
```

Comment:
In almost all of Hono (except for the JDBC packages), `Future.compose` instead of `Future.flatMap` is used (`flatMap` is an alias for `compose` here). Therefore I would prefer that here for better readability.

Reply:
Done.